packages/devextreme-angular/src/ui/popup/service/service.component.ts

```diff
@@ -28,12 +28,12 @@ import { DxServicePopupInsertionDirective } from './insertion.directive';
   ],
   template: '<ng-template popup-content-insertion></ng-template>',
 })
-export class DxServicePopupComponent extends DxPopupComponent implements AfterViewInit {
+export class DxPopupServiceComponent<T> extends DxPopupComponent implements AfterViewInit {
```

Comment:
Don't name is as another type to avoid confusing in the future:
```suggestion
export class ServiceComponent<T> extends DxPopupComponent implements AfterViewInit {
```

Reply:
agree